common/eth2/src/types.rs

```diff
@@ -1008,15 +1010,42 @@ impl FromStr for Accept {
     type Err = String;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        match s {
-            "application/octet-stream" => Ok(Accept::Ssz),
-            "application/json" => Ok(Accept::Json),
-            "*/*" => Ok(Accept::Any),
-            _ => Err("accept header cannot be parsed.".to_string()),
-        }
+        let mimes = parse_accept(s);
```

Comment:
```suggestion
        let mimes = parse_accept(s)?;
```

Reply:
`make lint` would return errors which does not relate to my changes at all.

It is already some errors in `unstable` branch.

Reply:
Hm.. I saw github actions are working fine. Maybe it is because my clippy version problem. I need to check.

Reply:
It is because I used `nightly` by default instead of `stable`